Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/placeholders/complex/user.dfy

```diff
@@ -0,0 +1,10 @@
+// RUN: %baredafny run %args %s --target=java --build=%S/Build/build --input %S/wrappers.dfy --input %S/random.dfy --input %S/randomJava.dfy --input %S/Interop/__default.java --spill-target-code > "%t"
+// RUN: %baredafny run %args %s --build=%S/Build/build --input %S/wrappers.dfy --input %S/random.dfy --input %S/randomCSharp.dfy --input %S/Interop.cs --spill-target-code >> "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method Main() {
+  var x := DfyRandom.GetRandomNat(1000000);
+  var y := DfyRandom.GetRandomNat(1000000);
+  var areEqual := x == y;
+  print "areEqual: ", areEqual;
```

Comment:
This is a good case for using `expect` instead of printing and comparing, I think.

Also I don't love the idea of a test that by design could non-deterministically fail, no matter how small the probability. :) How about `expect 0 <= x < 1000000` instead?

Reply:
> Also I don't love the idea of a test that by design could non-deterministically fail

I changed the chance to 10^-30, so it'll fail after our planet disintegrates

Reply:
> This is a good case for using expect instead of printing and comparing, I think.

Not sure what the advantage is, but sure !